cli/resource.ts

```diff
@@ -60,6 +56,14 @@ export async function pushResource(
       },
     });
   } else {
+    if (typeof data.is_oauth !== "undefined") {
+      console.log(
+        colors.yellow(
+          "! is_oauth has been removed in newer versions. Ignoring.",
+        ),
+      );
+    }
+
```

Comment:
That's not true ?

Reply:
I can't find this change via git blame, but [the 1.50 type def](https://deno.land/x/windmill@v1.50.0/windmill-api/models/CreateResource.ts?source) has is_oauth and [the 1.55 type def](https://deno.land/x/windmill@v1.55.0/windmill-api/models/CreateResource.ts?source) does not.

Reply:
Not sure why blame refuses to show this, but found it
https://github.com/windmill-labs/windmill/commit/37967a795006c2eb4e8b218abb3d1b0525c17d5e#diff-8bb7b61ea90ab2e4962997447574c28d7844f2450b33ee30064ec99b5bd66a01L4369-L4370
was between 1.52 and 1.51 during OAuth refactor